README.md

```diff
@@ -14,3 +18,19 @@ The Tekton API enables functionality to be seperated from configuration (e.g. [P
 
 
 Using `tektoncd/triggers` in conjunction with `tektoncd/pipeline` enables you to easily create full-fledged CI/CD systems where the execution is defined **entirely** through Kubernetes resources. This repo draws inspiration from `Tekton`, but can used stand alone since `TriggerTemplates` can create any Kubernetes resource.
+
+## Want to start using Tekton Triggers
+
+Probably a bit early! In the meantime, consider
+[ramping up on Tekton Pipelines](https://github.com/tektoncd/pipeline/tree/master/docs)
+
+## Want to contribute
+
+Hooray!
+
+- See [CONTRIBUTING.md](CONTRIBUTING.md) for an overview of our processes
+- See [DEVELOPMENT.md](DEVELOPMENT.md) for how to get started
+- Look at our
+  [good first issues](https://github.com/tektoncd/pipeline/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+  and our
+  [help wanted issues](https://github.com/tektoncd/pipeline/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
```

Comment:
Should these links point to the `triggers` repo? (ex: `https://github.com/tektoncd/triggers/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22`) although we don't have issues with those labels right now 😅

Reply:
whoops, thanks for catching that! 😅